src/scene/scene.js

```diff
@@ -785,6 +809,98 @@ class Scene extends EventHandler {
         }
     }
 
+    updateSkyboxProjection() {
+
+        if (this._skyboxProjectionEnabled) {
+            const scope = this.device.scope;
+
+            const center = this._skyboxProjectionCenter;
+            const radius = this._skyboxProjectionRadius;
+            const domeElevation = this._skyboxProjectionElevation;
+            scope.resolve('projectedSkydomeCenter').setValue([
+                center.x,
+                center.y * radius,
+                center.z,
+                1.0
+            ]);
+
+            scope.resolve('projectedSkydomeDome').setValue([
+                center.x,
+                center.y + radius * domeElevation,
+                center.z,
+                radius * radius
+            ]);
+
+            scope.resolve('projectedSkydomePlane').setValue([
+                0,
+                1,
+                0,
+                -center.y
+            ]);
+        }
+    }
+
+    /**
+     * When enabled, the skybox will be projected into a sphere dome and a ground plane. Otherwise
+     * the skybox will be projected onto an infinite cube. Defaults to false.
+     *
+     * @type {boolean}
+     */
+    set skyboxProjectionEnabled(value) {
+        if (value !== this._skyboxProjectionEnabled) {
+            this._skyboxProjectionEnabled = value;
+            this.updateSkyboxProjection();
+            this.updateShaders = true;
+        }
+    }
+
+    get skyboxProjectionEnabled() {
+        return this._skyboxProjectionEnabled;
+    }
+
+    /**
+     * The center of the skybox projection. The y-component of the vector represents the height of
+     * the camera from the ground plane. Defaults to (0, 0, 0).
+     *
+     * @type {Vec3}
+     */
+    set skyboxProjectionCenter(value) {
+        this._skyboxProjectionCenter.copy(value);
+        this.updateSkyboxProjection();
+    }
+
+    get skyboxProjectionCenter() {
+        return this._skyboxProjectionCenter;
+    }
+
+    /**
+     * The elevation of the skybox projection - both the sphere and the plane parts. Defaults to 0.
```

Comment:
This description isn't too clear to me...

Reply:
It specifies the placement of the skybox sphere (not the plain) in Y.